WooCommerce/Classes/ViewRelated/ReusableViews/TableFooterView.swift

```diff
@@ -1,12 +1,14 @@
 import Foundation
 import UIKit
+import SafariServices
+
 
 // MARK: - TableFooterView
 //
-class TableFooterView: UIView {
+final class TableFooterView: UIView {
 
     @IBOutlet private var icon: UIImageView!
-    @IBOutlet private var footnote: UILabel!
+    @IBOutlet public var footnote: UITextView!
```

Comment:
nit: probably doesn't need to be `public` since the class is not `public` and meant to be used in the same module, its default `internal` should work?